common/src/main/java/whocraft/tardis_refined/client/TardisClientData.java

```diff
@@ -255,6 +297,11 @@ public void tickClientside() {
                 HumSoundManager.playHum(SoundEvent.createVariableRangeEvent(humEntry.getSound()));
             }
 
+            // Stops the hum sound if there's no fuel
+            if (isThisTardis && fuel == 0 && soundManager.isActive(HumSoundManager.getCurrentSound())) {
+                HumSoundManager.playHum(SoundEvents.EMPTY);
```

Comment:
We usually just set the volume to 0 so there's less stopping and starting of the sound system

Reply:
Will change
Is there a method or something in the hum manager?

Reply:
@Duzos No, just inside loopingsound itself